NEWS.md

```diff
@@ -1 +1,21 @@
-* Incremental Release Notes here.
+## 2020-13-03 v1.0.0
+
+Export bibliographic MARC records from SRS
+
+* MDEXP-20: Project Setup
```

Comment:
It would be good if these are arranged in descending order. 
Please see https://wiki.folio.org/pages/viewpage.action?spaceKey=FOLIJET&title=Acquisitions+-+Release+Procedures It has very good and useful details

Reply:
done